src/System.Reflection.Emit/tests/EnumBuilder/EnumBuilder.Properties.Tests.cs

```diff
@@ -30,5 +30,25 @@ public void Namespace()
             enumBuilder.AsType();
             Assert.Empty(enumBuilder.Namespace);
         }
+
+        [Fact]
+        public void IsArray()
+        {
+            EnumBuilder enumBuilder = Helpers.DynamicEnum(TypeAttributes.Public, typeof(int));
+            Assert.False(enumBuilder.IsArray);
+            Assert.False(enumBuilder.IsSZArray);
+            Type asType = enumBuilder.AsType();
+            Assert.False(asType.IsArray);
+            Assert.False(asType.IsSZArray);
+            Type arrType = enumBuilder.MakeArrayType();
+            Assert.True(arrType.IsArray);
+            Assert.True(arrType.IsSZArray);
+            arrType = enumBuilder.MakeArrayType(1);
+            Assert.True(arrType.IsArray);
+            Assert.False(arrType.IsSZArray);
+            arrType = enumBuilder.MakeArrayType(2);
+            Assert.True(arrType.IsArray);
+            Assert.False(arrType.IsSZArray);
+        }
```

Comment:
Nit: in all of these tests, can you add blank lines between the relevant sets of tests, e.g.
```C#
Type asType = enumBuilder.AsType();
Assert.False(asType.IsArray);
Assert.False(asType.IsSZArray);

Type arrType = enumBuilder.MakeArrayType();
Assert.True(arrType.IsArray);
Assert.True(arrType.IsSZArray);
```
I find it hard to read when it's all smushed together like this.